apps/web/src/screens/Dashboard/Sui.tsx

```diff
@@ -56,6 +58,13 @@ export const SuiDashboard: FC<Props> = () => {
 		appActions.showSendModal(Networks.sui);
 	};
 
+	const handleChangePrivateSetting = (next: boolean) => {
+		settingsActions.updateHiddenBalanceSettings({
+			_id,
+			hideBalance: next,
+		});
```

Comment:
we already now `_id` of the setting, don't ask for it again.
also, the name of the function quite confusing.

we can just use `settingsActions.setPrivacy(value: bool);`